dmoj/graders/bridged.py

```diff
@@ -54,14 +55,17 @@ def _launch_process(self, case):
         os.close(submission_stdout_pipe)
 
     def _interact_with_process(self, case, result, input):
-        output = case.output_data()
+        judge_output = case.output_data()
         self._interactor_time_limit = (self.handler_data.preprocessing_time or 0) + self.problem.time_limit
         self._interactor_memory_limit = self.handler_data.memory_limit or env['generator_memory_limit']
+        args = self.handler_data.args or contrib_modules[self.contrib_type].ContribModule.get_args_string()
 
-        with mktemp(input) as input_file, mktemp(output) as output_file:
+        with mktemp(input) as input_file, mktemp(judge_output) as judge_file:
+            args = shlex.split(
+                args.format(input=shlex.quote(input_file.name), output='', answer=shlex.quote(judge_file.name),)
```

Comment:
Doesn't this result in a blank argument? Isn't that a problem?

Reply:
Fixed by splitting into an args string for checker and one for interactor.
The one for default interactor doesn't have `{output}`, but reading the `testlib.h` documentation, they expect an `output` file, so made the testlib contrib module compliant.

This breaks backwards compatibility with exactly 1 interactor in production, but by passing `args: '{input}, {answer}'` in its init.yml, it is resolved.